services/rfid/lib/rfid.js

```diff
@@ -0,0 +1,88 @@
+let mfrc522 = null;
+
+const DEBUG = process.env.DEBUG;
+
+// Try to handle not running on a Pi more gracefully
```

Comment:
Is there a use in running this when the RFID library can't be found? The stubbing seems a bit redundant.

Reply:
I've mocked the library to emit a UID for testing. But yeah, maybe not much point without an interactive way to customise the fake UIDs being published.

Reply:
`DEBUG` could send out a message of a test read over websockets?